assets/scenes/load_scene_example.scn.ron

```diff
@@ -7,10 +7,7 @@
   entities: {
     4294967296: (
       components: {
-        "bevy_ecs::name::Name": (
-          hash: 17588334858059901562,
-          name: "joe",
-        ),
+        "bevy_ecs::name::Name": "name",
```

Comment:
could you use a name that is not "name", just to avoid confusion? 🙂

Reply:
Oops, nice catch!